SupEngine/src/Input.ts

```diff
@@ -177,7 +179,7 @@ export default class Input extends EventEmitter {
     // Mouse
     this.newScrollDelta = 0;
     for (let i = 0; i <= 6; i++) {
-      this.mouseButtons[i] = { isDown: false, wasJustPressed: false, wasJustReleased: false };
+      this.mouseButtons[i] = { isDown: false,dblClicked: false, wasJustPressed: false, wasJustReleased: false };

```

Comment:
Missing a space before `dblClicked`. Should probably be `doubleClicked`, too (even if the event name is `dblclick`)